spec/create_facility_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+RSpec.describe CreateFacility do
+    before(:each) do
+        @new_facility = CreateFacility.new
+    end
+
+    describe '#initialize' do
+        it 'can initialize the class' do
+            expect(@new_facility).to be_an_instance_of(CreateFacility)
+        end
+    end
+
+    describe '#co_dmv_office_locations' do
+        it 'can create the co dmv office locations' do
+            co_dmv_office_locations = DmvDataService.new.co_dmv_office_locations
+            @new_facility.create_facilities(co_dmv_office_locations)
```

Comment:
Need an assert statement here!! The setup and execution of these tests is right on, but you need to assert something to actually have a full test.